components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -232,6 +253,17 @@ BraveProfileSyncServiceImpl::~BraveProfileSyncServiceImpl() {
   network_connection_tracker_->RemoveNetworkConnectionObserver(this);
 }
 
+bool BraveProfileSyncServiceImpl::IsSyncSetupIncomplete() const {
+  auto devices = brave_sync_prefs_->GetSyncDevices();
+  if (devices->size() < 2 &&
```

Comment:
sync setup is not complete is more than one device?

Reply:
yes, sync setup is not complete when have less than 2 devices